chrome/content/zotero/zoteroPane.js

```diff
@@ -1044,8 +1056,7 @@ var ZoteroPane = new function()
 	}
 	
 	function handleKeyUp(event) {
-		var from = event.originalTarget.id;
-		if (ZoteroPane.itemsView && from == ZoteroPane.itemsView.id) {
+		if (ZoteroPane.itemsView) {
```

Comment:
Probably don't need this at all, right? This was only there because we used it in the comparison.

Reply:
Oh yeah, that's right. I just force-pushed a commit without it